src/smallestPrimeFactor.js

```diff
@@ -12,14 +12,27 @@
  * @returns {number} The smallest prime factor of num
  */
 function smallestPrimeFactor(num) {
-  // This is your job. :)
+  primeFactors = [3, 5, 7, 11, 13]
+  let smallestPrimeFactorTwo = 2
+
+  if (num % 2 === 0) {
+    return smallestPrimeFactorTwo
+  }
+  else {
```

Comment:
Since the other branch `return`s if called, you don't need `else` here. It makes the code a little cleaner.